static/css/components/ui-tabs.less

```diff
@@ -43,6 +43,11 @@
         text-decoration: underline;
       }
     }
+    a:hover,
```

Comment:
This adds some additional CSS

I think in this file it makes sense to wrap the rule with 
```
/* stylelint-disable no-descending-specificity */ 
a:hover...
/* stylelint-enable no-descending-specificity */ 
```
and leave this file unchanged.